### PR TITLE
Fix an oversight wiith woodstock shotgun not being access locked

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About The Pull Request
augh

also update the gold m1911 examine text
## Why It's Good For The Game
fix good + flavours
## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

</details>

## Changelog
:cl:
fix: shotgun not access locked
spellcheck: Kayian M1911 updated description
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/misc/packs.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 	desc = "A classic Shotgun used by hunters, police and frontiersmen alike, now at an affordable price."
 	cost = PAYCHECK_COMMAND * 20
 	contains = list(/obj/item/gun/ballistic/shotgun/riot, /obj/item/storage/pouch/ammo, /obj/item/storage/belt/bandolier, /obj/item/ammo_box/advanced/s12gauge/hunter)
+	access_view = ACCESS_WEAPONS
 
 /datum/supply_pack/goody/rensoter_shotgun
 	name = "Renoster Shotgun Single-Pack"
```
Comment (Majkl-J) [inline modular_skyrat/modules/sec_haul/code/misc/packs.dm:44]: The other shotguns should also be limited to having a permit

**Review by Majkl-J — APPROVED** [review]

**Comment by projectkepler-ru** [comment]
Done

File: modular_skyrat/modules/sec_haul/code/misc/packs.dm [hunk 2]
```diff
@@ -38,12 +38,14 @@
...
 	desc = "A common modern shotgun used by Solarian State Police."
 	contains = list(/obj/item/gun/ballistic/shotgun/riot/sol, /obj/item/storage/pouch/ammo, /obj/item/storage/belt/bandolier, /obj/item/ammo_box/advanced/s12gauge/rubber)
 	cost = PAYCHECK_COMMAND * 22
+	access_view = ACCESS_WEAPONS
 
 /datum/supply_pack/goody/double_barrel
```
Comment (Majkl-J) [inline modular_skyrat/modules/sec_haul/code/misc/packs.dm:50]: The double shotty too

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/sec_haul/code/misc/packs.dm (hunk 2 above)
Comment (projectkepler-ru) [reply modular_skyrat/modules/sec_haul/code/misc/packs.dm:50]: It already is a TG weapon thats permit locked DUMMY

Comment (projectkepler-ru) [reply modular_skyrat/modules/sec_haul/code/misc/packs.dm:50]: Can you just get this overwith already

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**